### ISSUE TITLE
Unable to install/build with Sass following instructions in README

### ISSUE THREAD

**Issue by IrregularShed** [preactjs/preact-cli#1742]
**What is the current behaviour?**
Hi everyone. I'm trying to add Sass support to a new `preact-cli` created project. Following the instructions in the README no longer works, throwing an error in the `webpack-manifest-plugin`, which looks to involve the `tapable` module:

```
× ERROR TypeError: Cannot read properties of undefined (reading 'tap')
    at C:\dev\test\node_modules\webpack-manifest-plugin\dist\index.js:66:49
    at SyncHook.eval [as call] (eval at create (C:\dev\test\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:9:1)
    at SyncHook.lazyCompileHook (C:\dev\test\node_modules\tapable\lib\Hook.js:154:20)
    at Compiler.newCompilation (C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Compiler.js:630:30)
    at C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Compiler.js:667:29
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\dev\test\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:15:1)
    at AsyncSeriesHook.lazyCompileHook (C:\dev\test\node_modules\tapable\lib\Hook.js:154:20)
    at Compiler.compile (C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Compiler.js:662:28)
    at C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Watching.js:77:18
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\dev\test\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:33:1)
    at AsyncSeriesHook.lazyCompileHook (C:\dev\test\node_modules\tapable\lib\Hook.js:154:20)
    at Watching._go (C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Watching.js:41:32)
    at C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Watching.js:33:9
    at Compiler.readRecords (C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Compiler.js:529:11)
    at new Watching (C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Watching.js:30:17)
    at Compiler.watch (C:\dev\test\node_modules\preact-cli\node_modules\webpack\lib\Compiler.js:244:10)
```
Just before this error I'm getting a warning about peer dependencies between `@babel/eslint-parser` and `eslint` as well, although that _appears_ to be a separate issue.

**Steps to Reproduce**

1. Run `preact create` with all defaults
2. Make sure it's working with `npm run dev` - it will be - and exit the dev server
3. Follow the instructions in README: `npm install --save-dev sass sass-loader@10`
4. Rename and change one of the CSS files to an SCSS file to make sure there's something for it to load. The default `\src\components\header\style.css` file is a good one to do because you can nest the properties in a normal manner. (Remember to change `index.js` as well, to point to the newly named SCSS file.)

**What is the expected behaviour?**
Not that ☝️ 😁 I should be able to compile and run my application with SCSS files being parsed.

**Please mention any other relevant information**

```
Environment Info:
  System:
    OS: Windows 10 10.0.19044
    CPU: (4) x64 Intel(R) Core(TM) i5-4570 CPU @ 3.20GHz     
  Binaries:
    Node: 16.17.0 - C:\Program Files\nodejs\node.EXE
    npm: 8.15.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 105.0.5195.127
    Edge: Spartan (44.19041.1266.0), Chromium (105.0.1343.50)
  npmPackages:
    preact: ^10.3.2 => 10.11.0
    preact-cli: ^3.0.0 => 3.4.1
    preact-render-to-string: ^5.1.4 => 5.2.4
    preact-router: ^3.2.1 => 3.2.1
```

**Comment by rschristian** [comment]
I'm not able to reproduce, and it seems like you're missing a step in the reproduction? Are you running a build or dev server?

**Comment by IrregularShed** [comment]
At the moment I'm running a dev server. If you can't reproduce I'll try again in a clean VM, in case I've got something hanging around from old projects.

**Comment by IrregularShed** [comment]
OK, I created the app from within a Docker container and adding Sass works. Definitely something funky on my PC for me to work out.

Separately I had issues with this error: `error:0308010C:digital envelope routines::unsupported` which I fixed thanks to this blog post: https://bobbyhadz.com/blog/react-error-digital-envelope-routines-unsupported - I'm mentioning this purely for people like me who search repositories before asking questions!
